src/metadata/persistence/templates.clj

```diff
@@ -263,3 +263,14 @@
           (set-fields (prepare-template-deletion user))
           (where {:id template-id}))
   nil)
+
+(defn- used-attribute-subselect
+  []
+  (subselect [:template_attrs :ta] (where {:ta.attribute_id :attributes.id})))
```

Comment:
I know we're not using `attr_synonyms` currently, so it's probably not worth the time to account for those now, but maybe we should add a `TODO: also check attr_synonyms when those are used` comment to this function for future-us? 🚀 🤖

Reply:
Ah, good idea. Future us might appreciate that.